plugin/pkg/scheduler/factory/factory.go

```diff
@@ -191,6 +195,11 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys util.StringSe
 	// Cache this locally.
 	cache.NewReflector(f.createServiceLW(), &api.Service{}, f.ServiceLister.Store, 0).RunUntil(f.StopEverything)
 
+	// Watch and cache all ReplicationController objects. Scheduler needs to find all pods
```

Comment:
I think the second sentence of this comment is no longer accurate after you changed the way the algorithm works.